tests/src/test/scala/cats/tests/OptionTests.scala

```diff
@@ -76,4 +76,11 @@ class OptionTests extends CatsSuite {
       isEq.lhs should === (isEq.rhs)
     }
   }
+
+  // a test for OptionIdOps.some to always return Some
+
+  test("OptionIdOps.some"){
+    val s: String = null
+    s.some.contains(s) should === (true)
```

Comment:
Thanks, @vpavkin!

Would you mind leaving a comment in the code itself about why it's done this way?

Also, would you mind naming the test something that better identifies that it's checking for behavior when `null` is passed in? Maybe something like `test(".some with null argument still results in Some #871")`.